deps/v8/BUILD.gn

```diff
@@ -870,6 +870,8 @@ action("postmortem-metadata") {
     "src/objects/code.h",
     "src/objects/data-handler.h",
     "src/objects/data-handler-inl.h",
+    "src/objects/descriptor-array.h",
+    "src/objects/descriptor-array-inl.h",
```

Comment:
don't we need the same change in `v8.gyp`?

Reply:
I thought so, but it worked on my machine when I built it (metadata was correct in the final binary). I also couldn't find where to change on V8.gyp when I searched using grep. I think we're inferring this parameter from BUILD.gn now?

Reply:
Oh yes I'm sorry, it must be one of the lists that are scraped automatically from build.gn